RecoMET/METProducers/interface/HcalNoiseInfoProducer.h

```diff
@@ -110,6 +111,7 @@ namespace reco {
     edm::EDGetTokenT<reco::PFJetCollection> jet_token_;
 
     double TotalCalibCharge;    // placeholder to calculate total charge in calibration channels
+    double TotalLasmonCharge;    // placeholder to calculate total charge in laser monitor channels
```

Comment:
Class members should start with lowercase letters, uppercase initials are reserved to class names
(Please to the same also for TotalCalibCharge, and other possible occurrencies in this class)